core/coreunix/add.go

```diff
@@ -89,7 +89,7 @@ type Adder struct {
 	mroot      *mfs.Root
 	unlocker   bstore.Unlocker
 	tempRoot   *cid.Cid
-	Prefix     *cid.Prefix
+	Prefix     cid.Builder
```

Comment:
May want to call this `CidBuilder`.

Reply:
Okay.  Fixed. I was doing this quick and planned to clean it up later.